File: replace.py
import re
file_path = r"d:\program file\Project Kantor\e-arsip\resources\views\pages\laporan-spn\print.blade.php"

with open(file_path, "r", encoding="utf-8") as f:
    content = f.read()

# Replace both variants of the project name
content = re.sub(r"Pek\.? ?Perbaikan Bangunan Jaga Pintu Air Marina", r"{{ $payment->keperluan ?? '...' }}", content)

with open(file_path, "w", encoding="utf-8") as f:
    f.write(content)

print("Replacement successful!")

File: replace.php
<?php
$file = "d:/program file/Project Kantor/e-arsip/resources/views/pages/laporan-spn/print.blade.php";
$content = file_get_contents($file);

$content = preg_replace("/Pek\.?\s?Perbaikan Bangunan Jaga Pintu Air Marina/", "{{ \$payment->keperluan ?? '...' }}", $content);

file_put_contents($file, $content);
echo "Replacement successful!\n";
